cmd/bedMerge/bedMerge.go

```diff
@@ -6,11 +6,50 @@ import (
 	"flag"
 	"fmt"
 	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/numbers"
 	"log"
 )
 
-func bedMerge(infile string, outfile string, mergeAdjacent bool) {
+func bedMerge(infile string, outfile string, mergeAdjacent bool, lowMem bool) {
+	if lowMem {
+		bedMergeLowMem(infile, outfile, mergeAdjacent)
+	} else {
+		bedMergeHighMem(infile, outfile, mergeAdjacent)
```

Comment:
Is there an advantage to the highmem version?  Should the lowmem version replace the highmem version?

Reply:
I see now that highmem does not require the input to be sorted.  An idea for sometime in the future would be to add a "unSorted" flag where the program could read in the bed, sort it, and then send that slice to a channel that would run the lowmem code.  This might reduce our redundant code.